include/swift/AST/ASTBridging.h

```diff
@@ -1439,78 +1439,33 @@ void BridgedStmt_dump(BridgedStmt statement);
 //===----------------------------------------------------------------------===//
 
 // Bridged type attribute kinds, which mirror TypeAttrKind exactly.
-enum ENUM_EXTENSIBILITY_ATTR(closed) BridgedTypeAttrKind : size_t {
-  BridgedTypeAttrKind_autoclosure,
-  BridgedTypeAttrKind_convention,
-  BridgedTypeAttrKind_noescape,
-  BridgedTypeAttrKind_escaping,
-  BridgedTypeAttrKind_differentiable,
-  BridgedTypeAttrKind_noDerivative,
-  BridgedTypeAttrKind_async,
-  BridgedTypeAttrKind_Sendable,
-  BridgedTypeAttrKind_retroactive,
-  BridgedTypeAttrKind_unchecked,
-  BridgedTypeAttrKind_preconcurrency,
-  BridgedTypeAttrKind__local,
-  BridgedTypeAttrKind__noMetadata,
-  BridgedTypeAttrKind__opaqueReturnTypeOf,
-  BridgedTypeAttrKind_block_storage,
-  BridgedTypeAttrKind_box,
-  BridgedTypeAttrKind_dynamic_self,
-  BridgedTypeAttrKind_sil_weak,
-  BridgedTypeAttrKind_sil_unowned,
-  BridgedTypeAttrKind_sil_unmanaged,
-  BridgedTypeAttrKind_error,
-  BridgedTypeAttrKind_error_indirect,
-  BridgedTypeAttrKind_error_unowned,
-  BridgedTypeAttrKind_out,
-  BridgedTypeAttrKind_direct,
-  BridgedTypeAttrKind_in,
-  BridgedTypeAttrKind_inout,
-  BridgedTypeAttrKind_inout_aliasable,
-  BridgedTypeAttrKind_in_guaranteed,
-  BridgedTypeAttrKind_in_constant,
-  BridgedTypeAttrKind_pack_owned,
-  BridgedTypeAttrKind_pack_guaranteed,
-  BridgedTypeAttrKind_pack_inout,
-  BridgedTypeAttrKind_pack_out,
-  BridgedTypeAttrKind_owned,
-  BridgedTypeAttrKind_unowned_inner_pointer,
-  BridgedTypeAttrKind_guaranteed,
-  BridgedTypeAttrKind_autoreleased,
-  BridgedTypeAttrKind_callee_owned,
-  BridgedTypeAttrKind_callee_guaranteed,
-  BridgedTypeAttrKind_objc_metatype,
-  BridgedTypeAttrKind_opened,
-  BridgedTypeAttrKind_pack_element,
-  BridgedTypeAttrKind_pseudogeneric,
-  BridgedTypeAttrKind_unimplementable,
-  BridgedTypeAttrKind_yields,
-  BridgedTypeAttrKind_yield_once,
-  BridgedTypeAttrKind_yield_many,
-  BridgedTypeAttrKind_captures_generics,
-  BridgedTypeAttrKind_moveOnly,
-  BridgedTypeAttrKind_thin,
-  BridgedTypeAttrKind_thick,
-  BridgedTypeAttrKind_Count,
-  BridgedTypeAttrKind_isolated,
+enum ENUM_EXTENSIBILITY_ATTR(closed) BridgedTypeAttrKind {
+#define TYPE_ATTR(SPELLING, _) BridgedTypeAttrKind_##SPELLING,
+#include "swift/AST/Attr.def"
+  BridgedTypeAttrKind_None,
 };
 
 SWIFT_NAME("BridgedTypeAttrKind.init(from:)")
 BridgedTypeAttrKind BridgedTypeAttrKind_fromString(BridgedStringRef cStr);
 
-SWIFT_NAME("BridgedTypeAttributes.init(context:)")
-BridgedTypeAttributes BridgedTypeAttributes_create(BridgedASTContext cContext);
+SWIFT_NAME("BridgedTypeAttributes.init()")
+BridgedTypeAttributes BridgedTypeAttributes_create();
```

Comment:
I know this was the case before, but should we expose this to Swift as a static method named `new` or something to make it clear it's doing a manual allocation?